libs/designsystem/header/src/header.component.ts

```diff
@@ -93,17 +84,31 @@ export class HeaderComponent implements AfterContentInit, OnInit {
     return Array.isArray(this.subtitle2) ? this.subtitle2 : [this.subtitle2];
   }
 
-  onTitleClick(event: PointerEvent) {
+  public onTitleClick(event: PointerEvent) {
     this.titleClick.emit(event);
   }
 
-  _actionGroupInjector: Injector;
+  public _actionGroupInjector: Injector;
 
   private actionGroupConfig: ActionGroupConfig;
 
   constructor(private injector: Injector) {}
 
-  ngOnInit(): void {
+  public ngOnInit(): void {
+    this.initActionGroupInjector();
+
+    if (this.titleMaxLines > 0) {
+      this.fitHeadingConfig = {
+        maxLines: this.titleMaxLines,
+      };
+    }
+  }
+
+  public ngAfterContentChecked(): void {
```

Comment:
Are we sure we want to change this from `ngAfterContentInit` to `ngAfterContentChecked `? 🤔 Is it needed for anything specific?